### DISCUSSION TITLE
Change pm2 start into tcp4, not tcp6

### DISCUSSION THREAD

**Discussion by mrzdtydlntm** [TerriaJS/terriajs#5958] [Q&A] [answered]
Hello,
I'm Mirza from Indonesia. I want to ask something about starting terria. If I'm start with npm start, and I check it in netstat -tulnp, it started at tcp6. I want to start it in TCP. Does anyone know how to solve it?

Thank you for the answer.

Mirza.

**Comment by hardreddata** [comment, marked as answer]
Maybe https://stackoverflow.com/questions/60131421/how-to-force-node-js-pm2-to-run-based-on-ipv4 though I am no expert

**Reply by mrzdtydlntm** [reply]
Hello RusselGrew!
Thanks for your answer. I'll try to do it.
